onboarding_project/squad_pantry_app/models.py

```diff
@@ -98,3 +117,11 @@ class OrderDishRelation(models.Model):
 
     class Meta:
         unique_together = ["order", "dish"]
+
+
+class ConfigurationSettings(models.Model):
+    constant = models.CharField(max_length=256, unique=True)
+    value = models.IntegerField()
```

Comment:
`InteferField()`? What if I want to store a text in another settting?

Reply:
Done!